src/HealthCheck.Core/Results/HealthCheckResult.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HealthCheck.Core.Results
```

Comment:
I would like `HealthCheckResult` to remain in the `HealthCheck.Core` namespace. There won't be many different types of results, so introducing a new namespace feels unnecessary.

Reply:
frankly speaking i dont consider keeping bunch of files on same folder as a good practice but will not argue with you on this matter. Will change)